### PR TITLE
DOCS-1349: Add kubeNodePortRanges to the Felix table

### PR DESCRIPTION
**Pull request by girishshankaran**
Product Version(s):

Issue:
(https://tigera.atlassian.net/browse/DOCS-1349)

Link to docs preview:

SME review:
- [ ] An SME has approved this change. 

DOCS review:
- [ ] A member of the docs team has approved this change.

Additional information:

### CODE REVIEW

**Comment by girishshankaran** [comment]
@fasaxc Can you please review this one?

File: calico/reference/resources/felixconfig.mdx [hunk 1]
```diff
@@ -64,6 +64,7 @@ spec:
...
...ng in IPVS mode.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | string                                                                                                    | Comma-delimited list of port ranges     | `30000:32767`                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```
Comment (fasaxc) [inline calico/reference/resources/felixconfig.mdx]:
```suggestion
| kubeNodePortRanges                 | This parameter holds a list of port ranges used for service node ports. Only used if felix detects kube-proxy running in IPVS mode.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | string                                                                                                    | Comma-delimited list of port ranges     | `30000:32767`                                                                                                                                                                                                                                                                                                                                                                                                                                   |
```
Naming convention is different in the felixconfig resource.

Comment (fasaxc) [reply calico/reference/resources/felixconfig.mdx]: Oh, it's not even a comma-delimited string there either; it's a list of port ranges.

Comment (fasaxc) [reply calico/reference/resources/felixconfig.mdx]: https://github.com/projectcalico/calico/blob/master/api/pkg/apis/projectcalico/v3/felixconfig.go#L296

Comment (girishshankaran) [reply calico/reference/resources/felixconfig.mdx]: Changed "Comma-delimted list of port ranges" to "List of port ranges"
Changed "KubeNodePortRanges" to "kubeNodePortRanges"

Comment (girishshankaran) [reply calico/reference/resources/felixconfig.mdx]: Do you want me to make these changes in the older versions as well? 3.23 and 3.24?

Comment (girishshankaran) [reply calico/reference/resources/felixconfig.mdx]: @fasaxc Are these changes fine?

File: calico/reference/resources/felixconfig.mdx [hunk 2]
```diff
@@ -64,6 +64,7 @@ spec:
...
...ng in IPVS mode.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | string                                                                                                  | A list of port ranges                   | `30000:32767`                                                                                                                                                                                                                                                                                                                                                                                                                                     |
```
Comment (fasaxc) [inline calico/reference/resources/felixconfig.mdx]: It's not _a_ string, it's a _list of_ strings.

Comment (girishshankaran) [reply calico/reference/resources/felixconfig.mdx]: Fixed

Comment (girishshankaran) [reply calico/reference/resources/felixconfig.mdx]: @fasaxc Can you please approve this PR if you do not have any further comments?

**Review by fasaxc — APPROVED** [review]

**Pull request merged**